docs/quickstart/quickstart_chains/polkadot-moonbeam.md

```diff
@@ -0,0 +1,301 @@
+
+```ts
+export async function erc20Transfer(event: MoonbeamEvent<[string, string, BigNumber] & { from: string, to: string, value: BigNumber, }>): Promise<void> {
+    //We added a logger to the top of this function, in order to see the block number of the event we are processing.
+    logger.info(`Processing MoonbeamEvent at ${event.blockNumber.toString()}`);
+    const transfer = Erc20Transfer.create({
+        id: event.transactionHash,
+        from: event.args.from,
+        to: event.args.to,
+        amount: event.args.value.toBigInt(),
+        contractAddress: event.address,
+    });
+
+    await transfer.save();
+}
+```
+
+The `handleErc20Transfer` function receives event data from the EVM execution environment whenever an event matches the filters that was specified previously in the `project.yaml`. It instantiates a new `Transfer` entity and populates the fields with data from the EVM Call payload. Then the `.save()` function is used to save the new entity (_SubQuery will automatically save this to the database_).
+
+Check out our mappings documentation for [Substrate](../../build/mapping/polkadot.md) and the [Substrate WASM data processor](../../build/substrate-wasm.md) to get detailed information on mapping functions for each type.
+
+## 4. Build Your Project
+
+Next, build your work to run your new SubQuery project. Run the build command from the project's root directory as given here:
+
+::: code-tabs
+@tab:active yarn
+
+```shell
+yarn build
+```
+
+@tab npm
+
+```shell
+npm run-script build
+```
+
+:::
+
+::: warning Important
+Whenever you make changes to your mapping functions, make sure to rebuild your project.
+:::
+
+
+## 5. Run Your Project Locally with Docker
+
+SubQuery provides a Docker container to run projects very quickly and easily for development purposes.
+
+The `docker-compose.yml` file defines all the configurations that control how a SubQuery node runs. For a new project, which you have just initialised, you won't need to change anything.
+
+Run the following command under the project directory:
+
+::: code-tabs
+@tab:active yarn
+
+```shell
+yarn start:docker
+```
+
+@tab npm
+
+```shell
+npm run-script start:docker
+```
+
+:::
+
+::: tip
+It may take a few minutes to download the required images and start the various nodes and Postgres databases.
+:::
+
+Visit [Running SubQuery Locally](../../run_publish/run.md) to get more information on the file and the settings.
+
+## 6. Query Your Project
+
+There is a _Docs_ tab on the right side of the playground which should open a documentation drawer. This documentation is automatically generated and helps you find what entities and methods you can query. To learn more about the GraphQL Query language [here](../../run_publish/graphql.md).
+:::
+
+You should see results similar to below:
```

Comment:
You should give an example query script?

Reply:
Done